### PR TITLE
click event of button and window initialization

### PR DESCRIPTION
**Pull request by jyokush**

### CODE REVIEW

File: Connect.vb [hunk 1]
```diff
@@ -144,5 +149,81 @@ Public Class Connect
...
+    ''' <param name="toolWindowUserControl">
+    ''' Windows to host.
+    ''' </param>
+    Private Function CreateToolWindow(ByVal toolWindowCaption As String, ByVal toolWindowGuid As String,
+      ByVal toolWindowUserControl As UserControl) As Window
+        Dim userControlObject As Object = Nothing
+        Dim userControlHost As UserControlHost
+        Dim _SummaryWindow As Window
+        Dim progId As String
+        ' IMPORTANT: ensure that you use the same ProgId value used in the ProgId attribute of the UserControlHost
```
Comment (monikadhok) [inline Connect.vb]: Do not add such prefix in comments 'IMPORTANT'

File: Connect.vb [hunk 2]
```diff
@@ -144,5 +149,81 @@ Public Class Connect
...
+    ''' <param name="CancelDefault"></param>
+    Private Sub _Button_Click(Ctrl As Microsoft.Office.Core.CommandBarButton,
+      ByRef CancelDefault As Boolean) Handles _Button.Click
+        'If the button is clicked first time.
+        If count = 1 Then
+            'Make a copy of the file active in excel.
+            My.Computer.FileSystem.CopyFile(_VBE.ActiveVBProject.FileName, "D:\GoogleDrive.xlsm", True)
+            Dim letsTry As uploadFileToDrive = New uploadFileToDrive()
+            'Upload the file on drive.
+            fileId = letsTry.UploadFile("D:\GoogleDrive.xlsm")
```
Comment (monikadhok) [inline Connect.vb]: Add a TODO to handle non windows operating system.

Comment (monikadhok) [reply Connect.vb]: Use this when adding TODOs
https://github.com/apps/todo

File: Connect.vb [hunk 3]
```diff
@@ -144,5 +149,81 @@ Public Class Connect
...
+        progId = "MyVBAAddin.UserControlHost"
+        _SummaryWindow = _VBE.Windows.CreateToolWindow(_AddIn, progId, toolWindowCaption, toolWindowGuid, userControlObject)
+        userControlHost = DirectCast(userControlObject, UserControlHost)
+        _SummaryWindow.Visible = True
+        userControlHost.AddUserControl(toolWindowUserControl)
+        Return _SummaryWindow
+    End Function
+
+    ''' <summary>
+    ''' Implementation of the click event of the button.
```
Comment (monikadhok) [inline Connect.vb]: Which Button? Please be more specific.

File: Connect.vb [hunk 4]
```diff
@@ -144,5 +149,81 @@ Public Class Connect
...
+        End If
+        'if the data list count item is not zero then all Api in the file is not supported.
+        If lines.Count <> 0 Then
+            Dim userControlObject As Object = Nothing
+            Dim userControlToolWindow1 As UserControlToolWindow1
+            Try
+                    'If button is clicked the first time then window needs to initialize.
+                If _ApiSummaryWindow Is Nothing Then
+                    userControlToolWindow1 = New UserControlToolWindow1()
+                    _ApiSummaryWindow = CreateToolWindow("Report of API used in this project", "0EB93108-D229-4F6F-82C5-0B96AFFBB9C5", userControlToolWindow1)
```
Comment (monikadhok) [inline Connect.vb]: What is this 0EB93108-D229-4F6F-82C5-0B96AFFBB9C5?

File: README.md [hunk 5]
```diff
@@ -1,125 +1,4 @@
...
-* [Visual Studio](https://visualstudio.microsoft.com/vs/) - Vb.net Class Library Project
-* [Registry](https://stackoverflow.com/questions/1942626/build-add-in-for-vba-ide-using-vb-net) - Dependency Management
-
-## References
-
-* [Add-in](https://www.mztools.com/articles/2012/MZ2012013.aspx) - How to make add-in for VBA Editor
-* [Add-in](https://www.youtube.com/watch?v=y81Aq4bebZU) - YouTube link to make add-in project in Visual Studio
-* [Button](https://www.mztools.com/articles/2012/MZ2012015.aspx) - How to make different types of button in VBA Editor
-* [Tool Window](https://www.mztools.com/articles/2012/MZ2012017.aspx) - How to make Tool window in VBA Editor
+* The class which is used to create a Tool-Window is written in the `window` branch.
```
Comment (monikadhok) [inline README.md]: window branch? or window folder?

File: Connect.vb [hunk 6]
```diff
@@ -144,5 +149,81 @@ Public Class Connect
...
+      ByRef CancelDefault As Boolean) Handles _Button.Click
+        'If the button is clicked first time.
+        If count = 1 Then
+            'Make a copy of the file active in excel.
+            My.Computer.FileSystem.CopyFile(_VBE.ActiveVBProject.FileName, "D:\GoogleDrive.xlsm", True)
+            Dim letsTry As uploadFileToDrive = New uploadFileToDrive()
+            'Upload the file on drive.
+            fileId = letsTry.UploadFile("D:\GoogleDrive.xlsm")
+            'Delete that file.
+            My.Computer.FileSystem.DeleteFile("D:\GoogleDrive.xlsm")
```
Comment (monikadhok) [inline Connect.vb]: Please use string constants.

**Review by monikadhok — CHANGES_REQUESTED** [review]

File: Connect.vb (hunk 4 above)
Comment (jyokush) [reply Connect.vb]: Defined in CreateToolWindow() function summary, What does this argument mean.

File: Connect.vb (hunk 3 above)
Comment (jyokush) [reply Connect.vb]: Done.

File: Connect.vb (hunk 2 above)
Comment (jyokush) [reply Connect.vb]: Done.

File: Connect.vb (hunk 1 above)
Comment (jyokush) [reply Connect.vb]: Edited.

File: Connect.vb (hunk 6 above)
Comment (jyokush) [reply Connect.vb]: Done.

File: README.md (hunk 5 above)
Comment (jyokush) [reply README.md]: changed.

**Review by monikadhok — APPROVED** [review]

**Pull request merged**